### PR TITLE
add more information to the Azure setup

### PR DESCRIPTION
**Pull request by japrescott**
closes lig-533
- add more details to the Azure datasource documentation

### CODE REVIEW

File: docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst [hunk 1]
```diff
@@ -30,12 +30,12 @@ Preparing your data
...
 1. Install AzCopy cli by following the `guide of Azure <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10>`_
-2. To copy your data from the container to your local machine, go to "Security + networking > Shared access signature" on the storage account page in the Azure portal. Generate a shared access signature (SAS) which allows access to the container and objects.
-3. Copy the SAS token and use the following command:
+2. To sync your data from the container to your local machine (or vice versa), you need a shared access token. On your *storage account* page of the Azure portal, on the left, go to "Security + networking > Shared access signature". Generate a shared access signature (SAS) which allows access to the container and objects.
+3. Copy the SAS token and use the following command to sync the Azure blob storage with your local folder:
 
     .. code-block::
 
-        azcopy copy 'https://lightlydatalake.blob.core.windows.net/{YOUR_SAS_TOKEN}' '/local/lightlydatalake/farm-animals' --recursive
+        azcopy sync '/local/lightlydatalake/farm-animals' 'https://lightlydatalake.blob.core.windows.net/farm-animals{YOUR_SAS_TOKEN}' --recursive
```
Comment (philippmwirth) [inline docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst:38]: I think you're missing a / right before the YOUR_SAS_TOKEN

Comment (japrescott) [reply docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst:38]: the YOUR_SAS_TOKEN normally starts with a `?`. And the examples suggest that it doesnt need an `/`. However I can try and see what happens..

Comment (philippmwirth) [reply docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst:38]: Nono if it worked for you it's fine

**Review by philippmwirth — APPROVED** [review]

File: docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst (hunk 1 above)
Comment (japrescott) [reply docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst]: it works in both cases

**Pull request merged**